frontend/components/ClubPage/Events.js

```diff
@@ -0,0 +1,53 @@
+import { Icon, Text, StrongText, Card } from '../common'
+import { useState, useEffect } from 'react'
+
+const Events = props => {
```

Comment:
Would destructure props to start out
```suggestion
const Events =({ data }) => {
```